interface/HHReweight5D_Moriond17.h

```diff
@@ -0,0 +1,64 @@
+/*
+** class  : HHReweight5D
+** author : L. Cadamuro (LLR)
+** date   : 17 Feb 2017
+** brief  : Compute a reweight coefficient to transform an input distribution into a general point in kl, kt plane. 31/05/2017: Updated to 5D reweight
+*/ 
+
+
+#ifndef HHREWEIGHT_H
```

Comment:
I do not think that you're allowed to have two classes with same name defined here... don't you get errors while compiling the framework? I would change the name here in the old one...

Reply:
Indeed, this is true. 

Then I suggest we just do not keep the older version, if we will ever need it (which I think is unlikely to happen anyways) we will just brows back in the git history. I just don't want to waste time in modifying a class that we will never use again.

@camendola @francescobrivio what do you think?

Reply:
I'm in favour of removing whatever is obsolete. 

However, if @francescobrivio disagrees, I really don't think that the point @dzuolo raises is a problem. Only one class at a time is included in the skimmer, you don't get errors.
For example, cfr here where everything works out fine: 
[smallTree_HHbtag](https://github.com/LLRCMS/KLUBAnalysis/blob/VBF_legacy/interface/smallTree_HHbtag.h#L1-L2) [smallTree](https://github.com/LLRCMS/KLUBAnalysis/blob/VBF_legacy/interface/smallTree.h#L1-L2)

Reply:
I tried compiling and I get an error of the form:  `HHReweight5D_Moriond17.cc: multiple definition of 'HHReweight5D::getWeight()'; ../obj/HHReweight5D.o:HHReweight5D.cc: first defined here`

so apparently if you have to functions with identical and identical arguments within two classes with the same name, then you get in trouble and you cannot compile the executables, because in the shared library there are two identical functions

Reply:
ah right there is the .cc that is compiled. well, I don't know why I'm defending this since I want to remove the files `\_('')_/`

Reply:
If we want to stick to democracy 3 people over 4 are in favour of removing these files... :)